plugins/README.md

```diff
@@ -28,7 +28,7 @@ Read more about plugin authentication here: [Plugin authentication](https://plat
 ### Prerequisites
 
 1. The name of your plugin. This should be identical to the `NameForHuman` in your plugin manifest.
-   > Please refer to OpenAI for the [manifest requirements](https://platform.openai.com/docs/plugins/getting-started/plugin-manifest).
```

Comment:
What is wrong with this original link?

Reply:
The old link now redirects to the new one.

Reply:
The old one goes to a session under that page